modin/pandas/test/dataframe/test_reduce.py

```diff
@@ -181,6 +181,14 @@ def test_2195(datetime_is_numeric, has_numeric_column):
     )
 
 
+def test_describe_different_index():
+    modin_df = pd.DataFrame(np.zeros((2, 60)))
+    pandas_df = pandas.DataFrame(np.zeros((2, 60)))
+    modin_df["new"] = pandas_df["new"] = "abc"
```

Comment:
```suggestion
    pandas_df = pandas.DataFrame(test_data["int_data"])
    pandas_df["string_column"] = "abc"
    modin_df = pd.DataFrame(pandas_df)
```

`test_data['int_data']` has int data spanning multiple column partitions. Still, could you also please double check that the new test case reproduces the bug when you revert your fix?

Reply:
Yup, just checked that it failed without the fix. Printed stuff out and cross verified as well.